src/includes_normalize-win32.cc

```diff
@@ -96,8 +96,12 @@ string IncludesNormalize::Relativize(StringPiece path, const string& start) {
 
 string IncludesNormalize::Normalize(const string& input,
                                     const char* relative_to) {
-  char copy[_MAX_PATH];
+  char copy[_MAX_PATH + 1];
   size_t len = input.size();
+  if (len > _MAX_PATH) {
+    Warning("path too long '%s'\n", input.c_str());
+    return input;
```

Comment:
That seems fine also, up to you/Nico. One drawback is that I don't think we can do "death tests" with the non-gtest-test.h, but I could promote to an error return and print the message in the parent if that's preferable.